apps/OboeTester/app/src/main/cpp/analyzer/GlitchAnalyzer.h

```diff
@@ -169,8 +169,10 @@ class GlitchAnalyzer : public BaseSineAnalyzer {
                 break;
 
             case STATE_WAITING_FOR_LOCK:
-                mSinAccumulator += sample * sinf(mInputPhase);
-                mCosAccumulator += sample * cosf(mInputPhase);
+                mSinAccumulator += static_cast<double>(sample)
```

Comment:
An alternative is to make mSinAccumulator a float.

Reply:
Leaving as is because it's used later in division